doc/rst/language/spec/ranges.rst

```diff
@@ -591,7 +611,17 @@ values are interpreted as 0 or 1, respectively.  It is an error for
 the step to be zero.
 
 The type of the result of the ``by`` operator is the type of the base
-range, but with the ``stridable`` parameter set to ``true``.
+range, with the ``strides`` parameter updated according to the step.
+For example:
+
+- If the base range has ``strides=strideKind.one`` and the step is
+  the literal ``-2``, the result has ``strides=strideKind.positive``.
```

Comment:
Shouldn't this be negative?

Reply:
I started with `one`, so `positive` is definitely an improvement. :)